osu.Game/Online/API/Requests/Responses/BeatmapSetOnlineNominations.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using Newtonsoft.Json;
+
+namespace osu.Game.Online.API.Requests.Responses
+{
+    public struct BeatmapSetOnlineNominations
```

Comment:
The name should be in singular form, this is encapsulating a single nomination of one beatmap, not all nominations.